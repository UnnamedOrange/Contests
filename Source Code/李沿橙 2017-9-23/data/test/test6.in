5
7
1 1 1 100 1 1 1 
9
1 2 3 5 6 7 8 9 10 
7
1 3 7 9 15 16 17 
26
1 2 2 2 3 4 191 293 456 659 654 345 345 332 23 123 123 439 432 321 123 321 832 32 3 999 
8
1 2 3 4 7 8 9 10 
