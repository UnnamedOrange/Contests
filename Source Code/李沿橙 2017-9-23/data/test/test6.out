1 1 1 1 1 1 100 
1 2 5 7 8 10 9 6 3 
1 3 7 15 16 17 9 
1 2 2 2 3 3 4 23 32 123 123 123 191 293 321 321 345 345 432 439 456 654 832 999 659 332 
1 2 3 7 9 10 8 4 
